manager/controllers/app/blueprint_deploy_test.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 IBM Corp.
```

Comment:
Please rename to file `blueprint_controller_test.go` to align with the other controller tests

Reply:
@roee88 There is already a file with that name that does unit tests, which is why I had to choose a different name.